vaadin-themable-mixin.html

```diff
@@ -12,6 +12,18 @@
    */
   Vaadin.ThemableMixin = superClass => class VaadinThemableMixin extends superClass {
 
+    static get properties() {
+      return {
+        /**
+         * built-in variations in Element themes are applied using the theme attribute.
```

Comment:
I’d fix this as “Built-in variations of a component theme are applied using the `theme` attribute.”